L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -77,6 +77,9 @@
 from L1Trigger.L1CaloTrigger.l1EGammaEEProducer_cfi import *
 _phase2_siml1emulator.add(l1EGammaEEProducer)
 
+from Configuration.Eras.Modifier_phase2_trigger_cff import phase2_trigger
+phase2_trigger.toReplaceWith( SimL1EmulatorTask , _phase2_siml1emulator)
```

Comment:
should this be here?

Reply:
yes, it should.
The modules above this line do not depend on TrackTrigger tracks (i.e. on `phase2_trackerV14` modifier.)

Reply:
reusing `_phase2_siml1emulator` may be a problem here. you might want to make a separate `_phase2_siml1emulator_ttrack` where the subsequent modules are added.

Reply:
I could.
What would be a problem that you are suggesting ?

Reply:
By reusing the `_phase2_siml1emulator` object, the `phase2_trigger.toReplaceWith()` might include the items added to that object in subsequent lines. (But if this isn't the cause of the problems, then it should be fine.)